### PR TITLE
Create useDoubleEffect.ts

### PR DESCRIPTION
**Pull request by minnyww**

### CODE REVIEW

**Review by narze — CHANGES_REQUESTED** [review]
That's so useless 😂 
By the way please create usage component that use this hook (See readme)

**Comment by minnyww** [comment]
> 

alrady done

File: src/hooks-usage/useDoubleEffect.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export default function () {
+  const cardProps = {
+    desc: "useDoubleEffect - double your effect",
+    examples: [
+      {
+        code: `useDoubleEffect(() => {
+                console.log("Hi React");
+              }, []);`,
+        value: useDoubleEffect(() => {
+          "Hi react"
```
Comment (narze) [inline src/hooks-usage/useDoubleEffect.tsx]:
```suggestion
          console.log("Hi react")
```

**Review by narze — APPROVED** [review]
nice
nice

**Pull request merged**